co-simulation/bundle/src/assembly/resources/scenarios/Tiergarten/ns3/ns3_federate_config.xml

```diff
@@ -1,5 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <ns3>
+
+ <NetworkConfig>
+     <component name="CommType" value="LTE"/>
+     <component name="NumOfNodes" value="5"/>
```

Comment:
Does this determine the number of radios available or is that set somewhere else.

Reply:
The NumOfNodes variable in ns3_federate_config.xml is essential for predefining and initializing the C-V2X node. In the context of C-V2X mode, numerous NS-3 helpers are employed to configure the NS-3 node for C-V2X communication. However, unlike DSRC mode where nodes are dynamically created, some of these helpers require a setup at time=0. This necessity has been verified by the leader of the NS-3 development team. As a result, in C-V2X mode, the provision of the NumOfNodes variable is mandatory